### ISSUE TITLE
Dynamic descriptions and alerts are ready for Friction

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#81]
Please see the stripped down version of the A11y Design document:
https://docs.google.com/document/d/1v7GM_dNZs86jcRn2X03D2t7HFDb4dQ1h4ayDGXTu63U/edit#heading=h.uctn2leiizew

Assigning to @zepumph to review.
Please let me know if you want to meet and discuss.

**Comment by terracoda** [comment]
@zepumph, to reduce any confusion on my part or your part, I archived the Friction's A11y Main documetn and renamed the stripped down version to be the main one.

[Friction A11y Design Main](https://docs.google.com/document/d/1v7GM_dNZs86jcRn2X03D2t7HFDb4dQ1h4ayDGXTu63U/edit#heading=h.5uj8kow1p09i)
or (https://docs.google.com/document/d/1v7GM_dNZs86jcRn2X03D2t7HFDb4dQ1h4ayDGXTu63U/edit#heading=h.5uj8kow1p09i)

**Comment by terracoda** [comment]
@zepumph, when we are ready to implement the keyboard dialog, we can also implement the new content for all dialogs from this issue https://github.com/phetsims/a11y-research/issues/113#issuecomment-402547376

**Comment by zepumph** [comment]
Thoughts and comments so far:
* Should the thermometer be in the PDOM even though it isn't interactive?
* It may be hard to detect when the thermometer is "cooling down" so I haven't implemented that yet.
* I commented in the doc, but I also haven't added the "A" logic in the beginning sentence yet, as that will involve a bit more complexity as well.

I think that the PDOM content is pretty much done, and I will work more this afternoon on getting the alerts going.

**Comment by zepumph** [comment]
The above commit adds alerts when keyboard drag completes (not mouse drag right now). There alerts are not complete, and only cover as the simulation increases in friction.

**Comment by zepumph** [comment]
The big todos for this issue still are:
- [ ] Add alerts as the temp decreases (how do we trigger these alerts since the aren't from user input? Probably just through linking to the amplitude property)
- [ ] Adding directional alerts to the currently implemented alerts that will say "Left" or "Right' and then the quality of the atoms.
- [ ] Answer quite a few smallish questions, many of which are outlined in the above bullet list.

**Comment by terracoda** [comment]
@zepumph, re https://github.com/phetsims/friction/issues/81#issuecomment-409366781

> Should the thermometer be in the PDOM even though it isn't interactive?
@emily-phet and I discussed this and we decided for this sim to take a very simple structure. more akin to John Travoltage's structure. So for now the thermometer is included as part of the summary description.

> I cannot get the role description to be said on focus.
Currently, `aria-roledescription` is only recognized by VoiceOver. I was just re-reading the checklist (2.12 Custom Control Accessible Development Checklist:) in [Using ARIA](https://www.w3.org/TR/using-aria/), and I think you are correct, a change is needed to get the accessible name to read out. `aria-label` should work, but we may need to adjust. We may need to use an element inside the div with role application? This will be tricky to get right.
```html
<div role=”application” tabindex=”0” aria-roledescription=”move in four directions” aria-label="Chemistry book">Chemistry Book</div>

<!--OR Maybe??-->

<div role=”application” tabindex=”0” aria-roledescription=”move in four directions” ><h3>Chemistry Book</h3></div>
```
Basically, I can't wait to try out what you got, and we can iterate from there.

> "temperature zones" works for me.

Re https://github.com/phetsims/friction/issues/81#issuecomment-409424292
> Add alerts as the temp decreases (how do we trigger these alerts since the aren't from user input? Probably just through linking to the amplitude property)

@zepumph, I agree, I thought the temperature would be directly tied to atom jiggling speed. Lack of user interaction leads to less jiggling and thus a cooling temperature as well.

**Comment by terracoda** [comment]
@zepumph, you have done a great job implementing. The PDOM and alerts are working as designed.

I have identified 3 high-level design issues which I need to iterate on:
1. Length of alerts. Alerts are too long.
2. Lack of a clear distinction between state info and alert info. This sim is so dynamic, I'm struggling with how to find a good mix between state info and what’s happening info (alerts).
3. Wondering if we need to indicate more explicitly what arrow keys do? Left/Right rub book and Up/Down affect pressure?

I'll make iterations in design document. We may be able to close this issue.

**Comment by emily-phet** [comment]
@terracoda I'm going to go ahead and close this issue. Let's open up new ones for/as specific design and/or implementation challenges come up. 

(certainly reopen, though, if that seems most appropriate to you)

**Comment by terracoda** [comment]
@zepumph, just noticed in the A11y View that the break away alert is not uttered, only the second alert that contains the word "more" shows up in the Activity Log. I never see, "Very hot. Atoms break away from chemistry book." I only see, "Very Hot. More atoms break away from chemistry book."

I'll create an issue for that. I do think it will be useful to have logic for at least one (briefer) follow-up breakaway.
